app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Recipe:<%= @recipe.name %></h1>
+<h3>Complexity: <%= @recipe.complexity %></h3>
+<h3>Genre: <%= @recipe.genre %></h3>
+
+<h1>Ingredients:</h1>
+<% @ingredients.each do |ingredient| %>
+  <h3><%= ingredient.name %></h3>
+<% end %>
+
+<h1>Cost: <%= @cost %></h1>
+
+<%= form_with url: "/recipes/#{@recipe.id}", method: :post, data: { turbo: false } do |form| %>
```

Comment:
To make this even more explicit I would add the ingredients to your url:
```ruby
form_with url: "/recipes/#{@recipe.id}/ingredients" 
```
This just shows that you are creating a join with recipes and ingredients. So you would update your route to:
```ruby
post '/recipes/:id/ingredients', to: 'recipe_ingredients#create'
```